apps/backend/src/mcp/resources/compose_configs.py

```
@@ -81,7 +81,7 @@ async def get_compose_config_resource(uri: str) -> dict[str, Any]:
         return {
             "error": str(e),
             "uri": uri,
-            "timestamp": datetime.now(datetime.UTC).isoformat(),
+            "timestamp": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.12.2)

Undefined name `timezone`

(F821)